e2e/factories/fixtures/playwright.ts

```
@@ -0,0 +1,39 @@
+ */
+export const test = base.extend<{
+    factories: FactoryList;
+    factoryManager: FactoryManager;
+    stats: Record<string, number>
+}>({
+    factories: async ({}, use) => {
+        await factoryManager.setup();
+        await use(factoryManager.factories);
+    },
+
+    factoryManager: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)